Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py

```diff
@@ -705,17 +720,18 @@ def create_url_report_output(url, response, threshold, max_len, short_format):
     additional_info = response.get('additional_info', None)
     if additional_info is not None:
         resolution = additional_info.get('resolution', None)
-        if resolution is not None:
+        if resolution:
             md += 'IP address resolution for this domain is: ' + resolution + '\n'
-        if ec_url:
-            ec_url['VirusTotal'] = {
-                'Resolutions': resolution[:max_len]
-            }
-        else:
-            ec_url.update({
-                'VirusTotal': {'Resolutions': resolution[:max_len]},
-                'Data': url
-            })
+        update_entry_context_url(ec_url, url, field_name='Resolutions', field_value=resolution[:max_len])
```

Comment:
why are we trimming the resolution

Reply:
That's how it was before my fix I simply made the code more readable.
Looking at the code my best guess is that its doe for readability - @EdenOhana maybe you can shed some light here?